### PR TITLE
Rename GLC Greenland grid from "gland" to "gris"

### PR DESCRIPTION
**Pull request by billsacks**
We are renaming both the grid long name and alias to "gris". For now we
are maintaining backwards compatibility with old
aliases ("_gl4"/"_gl20").

Test suite: Ran a subset of tests from the aux_glc test list:
```
ERS_Ly11.f09_g17_gris20.T1850Gg.cheyenne_gnu
ERS_Vnuopc_Ly11.f09_g17_gris20.T1850Gg.cheyenne_gnu
ERS_Ly7.f09_g17_gris4.T1850Gg.cheyenne_intel
ERS_Vnuopc_Ly7.f09_g17_gris4.T1850Gg.cheyenne_intel
ERI.f10_f10_mg37.I1850Clm50SpG.cheyenne_gnu.cism-test_coupling
ERI_Vnuopc.f10_f10_mg37.I1850Clm50SpG.cheyenne_gnu.cism-test_coupling
SMS_D.T31_g37_gris20.I1850Clm50SpG.cheyenne_gnu.cism-test_coupling
SMS_Vnuopc_D.T31_g37_gris20.I1850Clm50SpG.cheyenne_gnu.cism-test_coupling
```

Test baseline: cism2_1_78
Test namelist changes: Yes, due to new grid name
Test status: bit for bit

Fixes none

User interface changes?: N

Update gh-pages html (Y/N)?: N

**Depends on cism changes in https://github.com/ESCOMP/CISM-wrapper/pull/58**

### CODE REVIEW

File: config/cesm/config_grids.xml [hunk 1]
```diff
@@ -450,35 +466,67 @@
       <mask>tn0.25v3</mask>
     </model_grid>
 
+    <model_grid alias="f09_g16_gris4" compset="_CISM">
```
Comment (mvertens) [inline config/cesm/config_grids.xml:469]: Are we really still supporting g16? I did not think so. Could you check if moving forwards we just remove this resolution. I am wondering at this point if it should totally be removed from config_grids.xml for cesm. At any rate - it certainly does not make sense to add a new item for CISM for it if we are no longer supporting it.

File: config/cesm/config_grids.xml [hunk 2]
```diff
@@ -450,35 +466,67 @@
...
     <model_grid alias="f09_g17_gl4" compset="_CISM">
       <grid name="atm">0.9x1.25</grid>
       <grid name="lnd">0.9x1.25</grid>
       <grid name="ocnice">gx1v7</grid>
-      <grid name="glc">gland4</grid>
+      <grid name="glc">gris4</grid>
       <mask>gx1v7</mask>
     </model_grid>
 
+    <model_grid alias="f09_g16_gris20" compset="_CISM">
```
Comment (mvertens) [inline config/cesm/config_grids.xml:501]: Same comment as above for g16.

File: config/cesm/config_grids.xml [hunk 3]
```diff
@@ -548,19 +596,35 @@
       <mask>gx1v7</mask>
     </model_grid>
 
+    <model_grid alias="f19_g16_gris4" compset="_CISM">
```
Comment (mvertens) [inline config/cesm/config_grids.xml:599]: Same comment for g16.

File: config/cesm/config_grids_common.xml [hunk 4]
```diff
@@ -105,32 +105,32 @@
     <!-- mapping is turned off in the system.)                    -->
     <!-- ======================================================== -->
 
-    <gridmap ocn_grid="gx1v6" glc_grid="gland4" >
+    <gridmap ocn_grid="gx1v6" glc_grid="gris4" >
```
Comment (mvertens) [inline config/cesm/config_grids_common.xml:108]: See comment for g16 above.

File: config/cesm/config_grids_common.xml [hunk 5]
```diff
@@ -105,32 +105,32 @@
...
     <!-- POP's estuary box model is currently not active for gx3v7, so
          we need nnsm maps for liquid as well as ice. -->
-    <gridmap ocn_grid="gx3v7" glc_grid="gland4" >
+    <gridmap ocn_grid="gx3v7" glc_grid="gris4" >
       <map name="GLC2OCN_LIQ_RMAPNAME">cpl/gridmaps/gland4km/map_gland4km_to_gx3v7_nnsm_e1000r500_180502.nc</map>
       <map name="GLC2OCN_ICE_RMAPNAME">cpl/gridmaps/gland4km/map_gland4km_to_gx3v7_nnsm_e1000r500_180502.nc</map>
     </gridmap>
 
-    <gridmap ocn_grid="gx1v6" glc_grid="gland20" >
+    <gridmap ocn_grid="gx1v6" glc_grid="gris20" >
```
Comment (mvertens) [inline config/cesm/config_grids_common.xml:123]: See comment for g16 above.

**Review by mvertens — CHANGES_REQUESTED** [review]

**Comment by billsacks** [comment]
A grep for "g16" across all CESM test lists shows 99 tests still using g16. Granted, these are mostly / all A, X, C and G compsets. My feeling was that it is easiest, both now and down the line, to just keep things consistent in this PR, and then we can remove all of the g16 in one fell swoop later. That said, if you prefer that I limit the additions of g16, a compromise could be avoiding adding new aliases involving g16 – but I think I still need to make some of the changes in this PR that refer to g16 until we are really ready to remove that grid. Let me know what you think.

**Comment by billsacks** [comment]
Okay, I just pushed a change where I removed some g16 grid aliases.

**Review by mvertens — APPROVED** [review]

**Comment by billsacks** [comment]
This cime change depends on the CISM changes in https://github.com/ESCOMP/CISM-wrapper/pull/58. I plan to bring that PR to cism master soon, but I think this cime PR should wait to come to master until we're ready within CESM to bring in the CISM changes, which might be a few weeks.

**Comment by jedwards4b** [comment]
@billsacks in that case I suggest using "Convert to draft"

**Comment by billsacks** [comment]
Converting to draft to signal that we aren't ready to merge this (though it's complete and ready to merge once we're ready on the CESM side).

**Pull request merged**